boa/src/context.rs

```diff
@@ -726,13 +725,14 @@ impl Context {
             .constructor(true)
             .build();
 
-        self.global_object().insert_property(
-            name,
+        self.realm.global_bindings.string_property_map_mut().insert(
```

Comment:
I would add some getters, things like `global_bindings()` to get the list of global bindings, and `global_bindings_mut()` to get the mutable reference, maybe directly to the context. Also, I would prefer if the `realm()` was not public.

Reply:
I also think the `Realm` should not be public, but let's leave that to a dedicated issue.

Reply:
Could you open the issue?